FitnessTracker.csproj

```diff
@@ -5,6 +5,9 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="AutoMapper" Version="9.0.0" />
+    <PackageReference Include="AutoMapper.Extensions.Microsoft.DependencyInjection" Version="7.0.0" />
```

Comment:
I like that you added automapper, but i didnt see you use it anywhere.  Maybe consider using automapper in some of the update methods of the repository